core/src/main/scala/kafka/log/LogCleanerManager.scala

```diff
@@ -64,6 +64,19 @@ private[log] class LogCleanerManager(val logDirs: Array[File], val logs: Pool[To
   @volatile private var dirtiestLogCleanableRatio = 0.0
   newGauge("max-dirty-percent", new Gauge[Int] { def value = (100 * dirtiestLogCleanableRatio).toInt })
 
+  /* a gauge for tracking the number of logCleaner runs */
+  @volatile private var numLogCleanRuns = 0
+  newGauge("num-runs", new Gauge[Int] { def value = numLogCleanRuns })
+
+  /* a gauge for tracking the time since the last log cleaner run, in milli seconds */
+  @volatile private var timeSinceLastRun : Long = System.currentTimeMillis
+  newGauge("time-since-last-run-ms", new Gauge[Long] { def value = System.currentTimeMillis - timeSinceLastRun })
+
+  /* a gauge for tracking the number of filthy logs. A non zero value indicates
+   * that the cleaner has not been successful in cleaning the logs. */
+  @volatile private var numFilthyLogs  = 0
+  newGauge("num-filthy-logs", new Gauge[Int] { def value = numFilthyLogs })
```

Comment:
This is really the number of logs currently being cleaned, right?  Filthy logs not being cleaned is not accounted for in this metric. Is this useful given the two other newly added metrics?